### PR TITLE
feat(plugin): Better Quests button

### PR DESCRIPTION
**Pull request by kvbaxi**
Requested in https://github.com/Vencord/plugin-requests/issues/361 by @TonyE2, **betterQuests** is a fairly simple plugin which introduces new button on the Home screen of the client - Quests
image

Said quests button functions as an easier/faster way to navigate through settings to Quests menu.

### CODE REVIEW

File: src/plugins/betterQuests/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                route={Routes.SETTINGS("inventory")}
+                icon={CrownIcon}
+            />
+        </ErrorBoundary>
+    );
+};
+
+export default definePlugin({
+    name: "BetterQuests",
+    description: "Puts the quest button in more accessibile place",
```
Comment (Grafaffel) [inline src/plugins/betterQuests/index.tsx:42]: Add punctuation to the description.

**Comment by TonyE2** [comment]
thx for doing my rquest

**Comment by kvbaxi** [comment]
> why wouldn't you put the actual quest menu in the tab instead of redirecting to settings 😭

I haven't thought about that when creating this plugin. I'll try to implement this now

**Comment by kvbaxi** [comment]
Added said quests section in home page instead of redirecting to inventory in settings and now the button has number badge indicating how many active/new quests you have.

**Pull request closed without merging**